ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -1,6 +1,6 @@
-<KvPageHeader @breadcrumbs={{@breadcrumbs}} @isEngineView={{true}} @pageTitle={{@backend}}>
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @mountName={{@backend}}>
   <:tabLinks>
-    <LinkTo @route={{@routeName}} data-test-secrets-tab={{@routeName}}>Secrets</LinkTo>
+    <LinkTo @route={{this.router.currentRoute.localName}} data-test-secrets-tab="Secrets">Secrets</LinkTo>
```

Comment:
I just discovered this router property the other day! I remember thinking, gosh I should use it more; and this is a great use case.